third_party/bazel/blade_helper/blade_helper_configure.bzl

```diff
@@ -1,11 +1,20 @@
-load("//bazel:common.bzl", "get_python_bin")
+load("//bazel:common.bzl", "get_python_bin", "get_host_environ")
 
 _PYTHON_BIN_PATH = "PYTHON_BIN_PATH"
+_BLADE_NEED_TENSORRT = "BLADE_WITH_TENSORRT"
+_BLADE_BUILD_INTERNAL = "BLADE_WITH_INTERNAL"
+_BLADE_BUILD_HIE = "BLADE_WITH_HIE"
 
 def _blade_helper_impl(repository_ctx):
     py_bin = get_python_bin(repository_ctx)
+    tensorrt_enabled = "True" if get_host_environ(repository_ctx, _BLADE_NEED_TENSORRT, "False").lower() in ["1", "true", "on"] else "False"
```

Comment:
A helper function for these boolean flags? Don't repeat yourself.

Reply:
actually for template replacement, it requires string values.